apps/settings/js/icc.js

```diff
@@ -119,6 +115,12 @@
 
     document.addEventListener('visibilitychange',
       visibilityChangeHandler, false);
+
+    iccStkMainHeader.addEventListener('action', function() {
+      if (typeof _backHandler === 'function') {
```

Comment:
I think we can just make the default value of `_backHandler` as a function, so that we don't have to check its type here every time when clicking on the header.